pilot/pkg/model/service.go

```diff
@@ -791,8 +791,11 @@ type TrafficDistribution int
 const (
 	// TrafficDistributionAny allows any destination
 	TrafficDistributionAny TrafficDistribution = iota
-	// TrafficDistributionPreferClose prefers traffic in same region/zone/network if possible, with failover allowed.
-	TrafficDistributionPreferClose TrafficDistribution = iota
+	// TrafficDistributionPrefer{Close,PreferSameZone} prefers traffic in same region/zone/network if possible, with failover allowed.
+	TrafficDistributionPreferClose    TrafficDistribution = iota
+	TrafficDistributionPreferSameZone TrafficDistribution = iota
```

Comment:
If these are the same we don't need 2 values for them. WE can convert the 2 k8s ones into one common one internally

Reply:
@hzxuzhonghu